### PR TITLE
messaging: cds.User.privileged for received messages

### PR DESCRIPTION
**Pull request by David-Kunz**

### CODE REVIEW

File: node.js/messaging.md [hunk 1]
```diff
@@ -157,6 +157,9 @@ If you want to receive all messages without creating topic subscriptions, you ca
 messaging.on('*', async msg => { /*...*/ })
 ```
 
+::: tip
+In general, messages do not contain user information, hence `cds.context.user` is set to [`cds.User.privileged`](../node.js/authentication#privileged-user).
```
Comment (sjvans) [inline node.js/messaging.md:161]:
```suggestion
In general, messages do not contain user information but operate with a technical user (i.e., service instance credentials).
As a consequence, the user of the message processing context (i.e., `cds.context.user`) is set to [`cds.User.privileged`](../node.js/authentication#privileged-user) and, hence, any necessary authorization checks must be done in custom handlers.
```

Comment (sjvans) [reply node.js/messaging.md:161]: we use the credentials from the service instance binding

Comment (renejeglinsky) [reply node.js/messaging.md:161]: @David-Kunz ping :)

Comment (David-Kunz) [reply node.js/messaging.md:161]: @sjvans 
> we use the credentials from the service instance binding

which service instance? I think I would just skip that - it's just the privileged user

Comment (renejeglinsky) [reply node.js/messaging.md:161]: @sjvans Do you agree?

Comment (sjvans) [reply node.js/messaging.md:161]: sure, we can remove the `(i.e., service instance credentials)` part

Comment (renejeglinsky) [reply node.js/messaging.md:161]: Like this?
```suggestion
In general, messages do not contain user information but operate with a technical user. As a consequence, the user of the message processing context (`cds.context.user`) is set to [`cds.User.privileged`](/node.js/authentication#privileged-user) and, hence, any necessary authorization checks must be done in custom handlers.
```

**Review by smahati — APPROVED** [review]

**Pull request merged**